### DISCUSSION TITLE
MODBUS TCP FZ94

### DISCUSSION THREAD

**Discussion by virtual-alchemy** [artisan-roaster-scope/artisan#681] [Q&A] [answered]
I just received the new FZ94 EVO from coffee tech :)

I cannot configure my Macbook pro Big Sur, I bought the Belkin USB-C LAN adapter but I cannot configure artisan to see the roaster. Does anyone have an idea?
<img width="737" alt="Capture d’écran 2021-09-05 à 10 32 00" src="https://user-images.githubusercontent.com/39820710/132120660-30b04c60-e6e1-4bd1-b78c-8c0499746254.png">

<img width="965" alt="Capture d’écran 2021-09-05 à 10 34 46" src="https://user-images.githubusercontent.com/39820710/132120745-8c094f54-90b0-408f-a68e-2b2313a1d05b.png">

**Comment by poundy** [comment]
Can you confirm what version of Artisan you've used, and if you've used Artisan at all before? Did you just go through the normal machine setup wizard?
Can you also show us the page from Config --> Port.. as well? We need to check the IP address and setup in that page.

**Comment by virtual-alchemy** [comment]
Hello I use
Artisan 2.4.6 build 2 (f1b8e57) 
Mac Os Big Sur 11.5.2
I have been using Artsian with an FZ94 for how long
As I can not manage with the mac I have configured a Windows 10 with Parallel Desktop
and it works :)
the ip address 192.168.1.2 is therefore good and visible in this case
I followed your advice and gave the Network adapter a manual ip address and
I attributed Parallel that my network is belkin usb-c-Lan
I used the default settings in the list of Coffee Tech FZ94 EVO roasters

**Comment by virtual-alchemy** [comment]
<img width="1120" alt="Capture d’écran 2021-09-06 à 08 38 11" src="https://user-images.githubusercontent.com/39820710/132174757-ede51c22-1e94-4246-9455-dca6799e04bf.png">

**Reply by poundy** [reply]
ok surprising to me was that the "default" IP address the coffee roaster is expected to use, as documented on https://artisan-scope.org/machines/coffeetech/, is 192.168.1.27, so yours on 192.168.1.2 seemed strange. But if you've proven it works on Windows, problem solved? (I'm not a Mac person at all, so can't tell you any vagaries of the Mac environment)

**Comment by virtual-alchemy** [comment, marked as answer]
yes I confirm that my EVO is the address 192.168.1.2

I also conform to the whims of the mac ...


I managed to get it to work on mac, the problem is to have everything on the same networks
